gufe/proteincomponent.py

```diff
@@ -61,11 +79,11 @@ def from_dict(cls, d: dict):
         raise NotImplementedError()
 
     def __hash__(self):
-        return hash((self.name, Chem.MolToSequence(self._rdkit)))
+        return hash(self.name)
 
     def __eq__(self, other):
         return hash(self) == hash(other)
 
     @property
     def total_charge(self):
-        return Chem.GetFormalCharge(self._rdkit)
+        return 0
```

Comment:
There must be a way to get this, gimme 5.

Reply:
Nope won't work, in that cases return `None`? The total charge is just undefine for now.

```suggestion
        return None
```